src/infrastructure/db/base_repository.lua

```
@@ -0,0 +1,413 @@
+    return "", {}
+  end
+  
+  local where_parts = {}
+  local params = {}
+  
+  for field, value in pairs(conditions) do
+    if type(value) == "table" and value.operator then
+      -- Handle complex conditions like {operator = "LIKE", value = "%search%"}
+      table.insert(where_parts, field .. " " .. value.operator .. " ?")
+      table.insert(params, value.value)
+    elseif type(value) == "table" and value.in then
```

Luacheck (1.2.0)

[error] expected identifier near 'in'

(E011)